apps/frontend/src/pages/quiz/libs/components/balance-wheel/balance-wheel.tsx

```diff
@@ -35,7 +45,13 @@ const BalanceWheel: React.FC = () => {
 		return (): void => {
 			clearInterval(intervalId);
 		};
-	}, [handleUpdatePercentage]);
+	}, [handleUpdatePercentage, navigate]);
+
+	if (percentage >= PercentageConfig.MAX_VALUE) {
+		void dispatch(authActions.getAuthenticatedUser());
```

Comment:
The user can't access this screen without signing it, so we don't need to get user again

Reply:
Is there a way we can refresh the state? Upon sending it back to the route page, all the states are back to the initial. So with that, it sends me back to the quiz part again. What I did with this screen is get the authenticated user again so I can send back to the root page safely.

Reply:
Just update `hasAnsweredQuizQuestions`, not the whole user

Reply:
I got the idea no. Thank you!